ui/common/src/userLink.ts

```diff
@@ -72,7 +72,7 @@ export const userTitle = (u: HasTitle): VNode | undefined =>
 export const fullName = (u: AnyUser) => [userTitle(u), u.name, userFlair(u)];
 
 export const userRating = (u: HasRating): string | undefined =>
-  u.rating ? ` (${u.rating + (u.provisional ? '?' : '')})` : undefined;
+  u.rating ? ` ${u.rating + (u.provisional ? '?' : '')}` : undefined;
```

Comment:
I think this removes the parens from every user link made by JS over the entire website.

Reply:
oops, you're right. I don't know where we have braces or not previously, so I guess I'll introduce a setting.